kr.md

```diff
@@ -1 +0,0 @@
-be curious not judgmental - Walt Whitman
```

Comment:
Whoops, didn't mean to delete! Let's fix!

Reply:
Thank you :)